### PR TITLE
Edit Profile for Mobile App

### PR DESCRIPTION
**Pull request by aemresafak**
I have implemented the edit profile screen, albeit being late for a week. I am very sorry about the delay. This PR concludes #512. Now users can edit their profiles.

### CODE REVIEW

File: prediction-polls/android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -15,9 +24,9 @@
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
-        android:usesCleartextTraffic="true"
         android:supportsRtl="true"
         android:theme="@style/Theme.PredictionPolls"
+        android:usesCleartextTraffic="true"
```
Comment (YigitSekerci) [inline prediction-polls/android/app/src/main/AndroidManifest.xml:29]: We need to think about this flag before the final milestone. Let's talk with BE team tomorrow.

**Review by YigitSekerci — CHANGES_REQUESTED** [review]
UI seems good in overall but you should connect this page to the backend. The backend is already implemented.

**Comment by aemresafak** [comment]
can you check again please yigit?

File: prediction-polls/android/app/src/main/java/com/bounswe/predictionpolls/data/profile/model/ProfileInfoResponse.kt [hunk 2]
```diff
@@ -5,7 +5,12 @@ import com.bounswe.predictionpolls.domain.profile.ProfileInfo
 import com.google.gson.annotations.SerializedName
 import kotlinx.collections.immutable.persistentListOf
 
-
+/*
+  "points": 0,
+  "biography": "string",
+  "birthday": "string",
+  "isHidden": 0
+ */
```
Comment (YigitSekerci) [inline prediction-polls/android/app/src/main/java/com/bounswe/predictionpolls/data/profile/model/ProfileInfoResponse.kt:13]: Nice touch Ahmet

**Review by YigitSekerci — APPROVED** [review]
Helal Ahmettt devamkee

**Pull request merged**